README.md

```diff
@@ -201,18 +171,12 @@ Welcome to
 
 #### BOOM WE'RE IN
 
-Now time for some cleanup.
-
-### Step 6: restoring the update script
+Now time for some cleanup. Make sure to be responsible now.
```

Comment:
Cleanup not necesarry with the SSH shadow file, so you can leave this out. Probably something I looked over :)

Reply:
Fixed in commit